### DISCUSSION TITLE
API Code Updating

### DISCUSSION THREAD

**Discussion by hamza-saqib** [bagisto/bagisto#6829] [Q&A]
I am using latest, version of API, and I want to make changes in API how i can do that. its located in vendor folder and I cant make changes there. what is the process of updating code for APIs ?

**Comment by Sanjaybhattwebkul** [comment]
Hi there,
Please follow [This](https://forums.bagisto.com/topic/2862/where-does-by-default-api-hits/6) thread.

https://forums.bagisto.com/topic/2862/where-does-by-default-api-hits/6
